lib/evmone/instructions.cpp

```diff
@@ -316,7 +316,7 @@ constexpr std::array<instruction_exec_fn, 256> instruction_implementations = [](
     table[OP_SHL] = op<shl>;
     table[OP_SHR] = op<shr>;
     table[OP_SAR] = op<sar>;
-    table[OP_EXTCODEHASH] = op<extcodehash>;
```

Comment:
Why is this removed here?

Reply:
Oh yeah, I was wandering the same @yperbasis.

Reply:
Oh, I simply removed duplicated code; see line 323.

Reply:
Ah ok. I will sort these in separate PR then.